src/App.js

```diff
@@ -2,14 +2,19 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Link, NavLink } from 'react-router-dom';
 import Home from './containers/Home'
 import Page from './containers/Page'
+import GamepadComponent from './components/GamepadComponent';
+import { Provider as ControllerProvider } from './context/Controller';
 
 export default function App() {
   return (
     <Router>
-    <div className="container">
-      <Route path="/" exact component={Home} />
-      <Route path="/page" component={Page} />
-    </div>
+      <ControllerProvider>
+        <div className="container">
+          <GamepadComponent />
```

Comment:
What do you think about putting GamepadComponent into one of the routes and that way we will also have routes of more stuff and not just a game pad

Reply:
I don't see the need for routes at this point###